tests/Backend/Snowflake/TypedTableInWorkspaceTest.php

```diff
@@ -119,6 +119,224 @@ public function testUnloadFromWSToTypedTable(): void
         $this->unloadAndAssert($workspace['id'], $tableId);
     }
 
+    public function testUnloadFromWSToTypedTableCTAS(): void
+    {
+        if (!in_array('ctas-om', $this->_client->verifyToken()['owner']['features'], true)) {
+            $this->markTestSkipped(
+                'CTAS is not enabled for this project, skipping test.',
+            );
+        }
+        // create workspace and source table in workspace
+        $workspace = $this->initTestWorkspace();
+
+        $tableId = 'Languages3';
+
+        $backend = WorkspaceBackendFactory::createWorkspaceBackend($workspace);
+        $backend->dropTableIfExists($tableId);
+
+        $connection = $workspace['connection'];
```

Comment:
wut? vzdyt to pripojeni natavuje schema ne?

Reply:
c&p z předchozího testu 🧌

Reply:
no dobre, byt to hotova vec, tak to po tobe chci jeste uparvit...